lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -5,6 +5,7 @@
             type='text'
             class='form-control'
             placeholder='{{t 'general.filter'}}'
+            oninput={{perform @manager.changeFilter value='target.value'}}
```

Comment:
Can we use the `<Input>` helper instead? We can bind the `value` directly to `this.filter` and use `{{on 'input'}}` element modifier to trigger `@manager.changeFilter`. I believe this is the more Octane-y (Octanic? Octanenous?) way.

Reply:
Hmmm. It's weird; `on 'input'` doesn't send an event when the field is fully cleared, and `on 'change'` only fires when enter is pressed or the focus is changed. I'm not seeing a way to make this work as well as it was before. Any ideas?

Reply:
No, I was misinterpreting what it was doing. It's always one input behind in the request. So it updates this.filter after it does the `on 'input'` rather than before. I'll keep looking into this, but if you happen to know what I've done wrong, please let me know.

Reply:
I ended up using `on 'keyup'` instead.